site/content/docs/5.2/components/navbar.md

```diff
@@ -127,7 +127,7 @@ The `.navbar-brand` can be used to contain most elements, but an anchor works be
 
 ### Nav
 
-Navbar navigation links build on our `.nav` options with their own modifier class and require the use of [toggler classes](#toggler) for proper responsive styling. **Navigation in navbars will also grow to occupy as much horizontal space as possible** to keep your navbar contents securely aligned.
```

Comment:
Should we re-introduce the content from Bootstrap with a callout instead ? (Only things that we don't want to show in Boosted)